identity-service/src/routes/instagram.js

```diff
@@ -161,9 +161,16 @@ module.exports = function (app) {
         const instagramObj = await models.InstagramUser.findOne({
           where: { uuid }
         })
+        const user = await models.User.findOne({
+          where: { handle }
+        })
 
+        const isUnassociated = instagramObj && !instagramObj.blockchainUserId
+        const handlesMatch =
+          instagramObj &&
+          instagramObj.profile.username.toLowerCase() === user.handle
```

Comment:
should `user.handle` be `.toLowerCase()` as well - no sure it it's already lowercased